pr.codecept.js

```diff
@@ -75,6 +75,8 @@ exports.config = {
   },
   mocha: {
     reporterOptions: {
+      reportDir: './tests/output',
```

Comment:
please test your changes on CI

Reply:
reverted and updated readme.https://pmm.cd.percona.com/blue/organizations/jenkins/pmm2-ui-tests/detail/pmm2-ui-tests/10318/pipeline/